.github/workflows/test-amazon-web-services-install.yaml

```diff
@@ -80,6 +91,7 @@ jobs:
         helm upgrade --install \
           -f ci/values/amazon_web_services.yaml \
           --set "ingress.hostname=${{ steps.vars.outputs.fqdn_record }}" \
+          --set "image.sha=${{ github.event.inputs.posthog-image-sha }}" \
```

Comment:
What will this value be on push events? It seems like it would break normal deploys.

Reply:
Yes was just getting this up to help with breaking the release, I’ll switch to draft pr 🙏